index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Odin-recipes</title>
+</head>
+
+<body>
+    <h1>Odin Recipes</h1>
+    <h3>Chef AATIQ</h3>
+    <img src="../images/chef.jpg">
+
+    <h4>List of Recipes</h4>
+    <ul>
+        <li>
+            <a href="recipes/biryani.html">Biryani</a>
+        </li>
+        <li>
+            <a href="recipes/currried-egg-sandwich.html">Curried Egg Sandwich</a>
+        </li>
+        <li>
+            <a href="recipes/chocolate-milk.html">Chocolate Milk</a>
+        </li>
+    </ul>
+
+</body>
+
+</html>
```

Comment:
Add an additional line at the end of the file. After raising a PR, always review it yourself and check if a red circle appears, indicating that you need to add a new line at the end of the file.